docs/animations/Expressions.md

```diff
@@ -0,0 +1,110 @@
+# ExpressionBuilder 
+Welcome to the ExpressionBuilder classes! The ExpressionBuilder classes are a C#-only alternative to building Expressions with type safety. Below is a quick introduction to using the ExpressionBuilder classes with your application. For complete documentation and walkthroughs, please see the Word Document in the project folder titled "ExpressionBuilder_Documentation.docx".
```

Comment:
Where is this ExpressionBuild_Documentation.docx? Should we have it as a markdown file in the documentation?